src/bindings/DatePicker.re

```diff
@@ -0,0 +1,26 @@
+[@genType]
+type state('date) = {value: array('date)};
+
+[@genType]
+type renderDateRangePicker('date) = {
+  date: 'date,
+  value: array('date),
+  onChange: unit => unit,
+};
+
+[@genType.import "@wojtekmaj/react-daterange-picker"]
+[@bs.module "./DatePicker.gen"]
+external make:
+  (
+    ~value: array('date),
+    ~onChange: state('date) => unit,
+    'a
+  ) =>
+  ReasonReact.component(
```

Comment:
this bind is not like this anymore, you just return now `React.component`
after rewriting DatePicker in ReasonReact you can just bind @wojtekmaj/react-daterange-picker itself

Reply:
eita, we have examples in another app?

Reply:
Here! https://github.com/Astrocoders/astro-design/blob/feature/popover/src/components/Popover.re#L10